### PR TITLE
Fixes markdown translator

### PR DESCRIPTION
**Pull request by tom-wa**
# Purpose

updated shell recorder + markdown translator syntax + added support for multiline commands (e.g. heredoc)

Syntax description in `tutorial_wrapper/README.md`

plugin examples updated to match new syntax

### CODE REVIEW

File: tests/shell/shell_recorder/tutorial_wrapper/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Markdown to Shell Recorder #
+
```
Comment (markus2330) [inline tests/shell/shell_recorder/tutorial_wrapper/README.md]: Please some general info what this is about.

File: tests/shell/shell_recorder/tutorial_wrapper/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Syntax ##
+
+* Checks
```
Comment (markus2330) [inline tests/shell/shell_recorder/tutorial_wrapper/README.md]: What does it check? How do I write a command?

File: tests/shell/shell_recorder/tutorial_wrapper/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+     compares the return code of the command to N. if not specified, 0 is implied.
+
+  * `# ERROR:N`
+
+     checks if the error with the code N was thrown
+
+  * `# WARNINGS:REGEX-STRING`
+
+     `REGEX-STRING` is matched against a comma-separated list of the warnings thrown by the command using posix-extended regex
```
Comment (markus2330) [inline tests/shell/shell_recorder/tutorial_wrapper/README.md]: Please give an example.

File: tests/shell/shell_recorder/tutorial_wrapper/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+* Comments
+
+  all lines starting with `#` that aren't checks are treated as comments and will be ignored
+
+* Empty lines
+
+  will be ignored
+
+* Commands
```
Comment (markus2330) [inline tests/shell/shell_recorder/tutorial_wrapper/README.md]: Should be in the beginning!

**Comment by markus2330** [comment]
Looks much better now! Thank you! Please set as ready to merge!

**Comment by tom-wa** [comment]
should i also implement a way to automatically unmount everything that has been mounted at the end ?

**Comment by markus2330** [comment]
Yes, that would be great, but maybe in a different PR, this seems to be quite finished?

Btw. it should not be limited to mounts, but actually it should undo everything that was done. So maybe just always make "Backup-and-Restore:/"? (globalplugins are another source of big problems if a testcase does not restore it properly)

**Comment by markus2330** [comment]
Thank you!

**Pull request merged**